test/harvest/ansible/roles/run-harvest-tests/tasks/execute-tests-Linux.yaml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: execute tests
+  shell: "{{ ansible_user_dir }}/{{ os_arch_binary_name }} -test.run=\"{{ tests_to_run_regex }}\" -test.v"
+  ignore_errors: true
```

Comment:
ignore errors and no log so all the output from harvest is not printed in one line. Instead capture the output and print later stderr only if it fails.